djstripe/templatetags/djstripe_tags.py

```diff
@@ -22,3 +22,20 @@ def djdiv(value, arg):
         except Exception:
             return ''
 division.is_safe = False
+
+
+@register.filter(name='djstripe_plan_level')
+def djstripe_plan_level(name):
+    """
+    Add support to levels over plans, then you can have different kind of plans with the level same access.
+
+    Use: {{ <plan_name>|djstripe_plan_level }}
+
+    Custom settings setup need it, please see the documentation for details.
```

Comment:
``Note: Custom settings setup is needed, please see the documentation for details.``